viam_cartographer.go

```diff
@@ -205,6 +217,10 @@ func New(
 		return nil, err
 	}
 
+	if optionalConfigParams.LidarDataRateMsec == 0 && optionalConfigParams.ImuDataRateMsec != 0 {
```

Comment:
[q] What if it is the other way around, i.e. when `LidarDataRateMsec != 0` and `ImuDataRateMsec == 0`?

Reply:
Good point!